resources/views/menu/admin_menu.blade.php

```diff
@@ -34,6 +41,36 @@
                 {
                     array_push($listItemClass,'active');
                 }
+
+                // Childrens Permission Checker
```

Comment:
I would really prefer to avoid duplicating this entire section... Not sure how best to approach that at the moment though.  Open for suggestions.